components/Table/TablePercent.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+import styles from './TablePercent.css';
+
```

Comment:
The trade off is that you have to pass all the props to the function (which can be annoying), though it does follow a more functional style since it becomes a pure function.